.eslintrc.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
```

Since we're using the `eslint-config-prettier` rather than the `eslint-plugin-prettier` we need to update the `extends` here:

```suggestion
  "extends": ["eslint:recommended", "prettier"],
```

---

done